bin/download_upload.sh

```diff
@@ -19,7 +19,9 @@
 # COMPRESSION_TYPE can be NONE or GZIP
 
 # download the file locally
-source /root/eggo/eggo-ec2-variables.sh
+if [ -f /root/eggo/eggo-ec2-variables.sh ]; then
+  source /root/eggo/eggo-ec2-variables.sh
+fi
```

Comment:
We may be able to eliminate this script entirely by using the Luigi Hadoop API.  I'll wait till we merge this PR before I try to refactor.